pkg/controller/controller_test.go

```diff
@@ -2202,6 +2215,21 @@ func assertServiceInstanceCredentialPropertiesStateParametersUnchanged(t *testin
 	}
 }
 
+func assertServiceInstanceCredentialOrphanMitigationSet(t *testing.T, obj runtime.Object, inProgress bool) {
+	binding, ok := obj.(*v1alpha1.ServiceInstanceCredential)
+	if !ok {
+		fatalf(t, "Couldn't convert object %+v into a *v1alpha1.ServiceInstanceCredential", obj)
+	}
+
+	if e, a := inProgress, binding.Status.OrphanMitigationInProgress; e != a {
+		if a {
```

Comment:
Can we collapse this into a single line that passes in the expected value to replace the hard-coded false and true?